saliency/blur_ig_test.py

```diff
@@ -0,0 +1,55 @@
+"""Tests completeness axiom for blur_ig.
+"""
+
+from absl.testing import absltest
+import numpy as np
+import tensorflow.compat.v1 as tf
+
+from . import blur_ig
+
+
+class BlurIgTest(absltest.TestCase):
+  """
+  To run:
+  "python -m saliency.blur_ig_test" from the PAIR-code/saliency directory.
+  """
+
+  def testBlurIGGetMask(self):
+    max_sigma = 10
+    with tf.Graph().as_default() as graph:
+      x = tf.placeholder(shape=[None, 5, 5, 1], dtype=tf.float32)
+      # Define function to just look at center pixel.
+      y = x[:, 2, 2, 0] * 1.0
+      with tf.Session() as sess:
+        # All black except 1 white pixel at the center.
+        x_input_val = np.array([[0.0, 0.0, 0.0, 0.0, 0.0],
+                                [0.0, 0.0, 0.0, 0.0, 0.0],
+                                [0.0, 0.0, 1.0, 0.0, 0.0],
+                                [0.0, 0.0, 0.0, 0.0, 0.0],
+                                [0.0, 0.0, 0.0, 0.0, 0.0],
+                               ], dtype=np.float)
+        x_input_val = x_input_val.reshape((5, 5, 1))
+        # Calculate the value of `y` at the input.
+        y_input_val = sess.run(y, feed_dict={x: [x_input_val]})
+
+        # Baseline is the fully blurred version of the input.
+        x_baseline_val = blur_ig._gaussian_blur(x_input_val, sigma=max_sigma)
```

Comment:
is gaussian_blur is something useful to end users, make it public API and documented it (e.g. remove the leading underscore)

Reply:
Good point. Have made it public. Thanks!